app/views/About.js

```diff
@@ -23,18 +25,60 @@ class AboutScreen extends Component {
   }
 
   handleBackPress = () => {
+    this.setState({ tapCount: 0 });
     this.backToMain();
     return true;
   };
 
   componentDidMount() {
+    GetStoreData(DEBUG_MODE).then(dbgMode => {
+      if (dbgMode == 'true') {
+        this.setState({ tapCount: 4 });
+      }
+    });
+
     BackHandler.addEventListener('hardwareBackPress', this.handleBackPress);
   }
 
   componentWillUnmount() {
     BackHandler.removeEventListener('hardwareBackPress', this.handleBackPress);
   }
 
+  handleTapTeam = () => {
+    this.setState({ tapCount: this.state.tapCount + 1 });
+    if (this.state.tapCount >= 3) {
+      if (this.state.tapCount == 3) {
+        // Debug mode on
+        SetStoreData(DEBUG_MODE, 'true');
+
+        // Create faux intersection data
+        let pseudoBin = [];
+        for (let i = 0; i < 28; i++) {
+          const intersections = Math.max(
+            0,
+            Math.floor(Math.random() * 50 - 20),
+          );
+          pseudoBin.push(intersections);
+        }
+        let dayBin = JSON.stringify(pseudoBin);
+        SetStoreData(CROSSED_PATHS, dayBin);
```

Comment:
Does this overwrite real data? perhaps we can store fake data in `CROSSED_PATHS_DEMO`

`GetStoreData(CROSSED_PATHS)` can check for `DEMO_MODE` and return `CROSSED_PATHS_DEMO` if true.. this would allow demo mode to work in all locations.

Reply:
Modified to restore data on exit.